includes/admin/customers/customers.php

```diff
@@ -597,12 +597,12 @@ function edd_customers_view( $customer = null ) {
 			<li>
 				<a href="<?php echo admin_url( 'edit.php?post_type=download&page=edd-payment-history&customer=' . $customer->id ); ?>">
 					<span class="dashicons dashicons-cart"></span>
-					<?php printf( _n( '%s Completed Sale', '%s Completed Sales', $customer->purchase_count, 'easy-digital-downloads' ), esc_html( number_format_i18n( $customer->purchase_count, 0 ) ) ); ?>
+					<?php printf( _n( '<span class="edd_purchase_count">%s</span> Completed Sale', '<span class="edd_purchase_count">%s</span> Completed Sales', $customer->purchase_count, 'easy-digital-downloads' ), esc_html( number_format_i18n( $customer->purchase_count, 0 ) ) ); ?>
```

Comment:
The HTML should wrap around the placeholder so translators do not accidentally remove it.

```suggestion
					<?php
					printf(
						_n(
							'%s Completed Sale',
							'%s Completed Sales',
							$customer->purchase_count,
							'easy-digital-downloads'
						),
						(
							'<span class="edd_purchase_count">' .
							esc_html( number_format_i18n( $customer->purchase_count, 0 ) ) .
							'</span>'
						)
					);
					?>
```